operator/src/main/java/oracle/kubernetes/operator/helpers/DomainValidationSteps.java

```diff
@@ -80,12 +79,12 @@ public NextAction onSuccess(Packet packet, CallResponse<V1SecretList> callRespon
     }
 
     static List<V1Secret> getSecrets(Packet packet) {
-      return Optional.ofNullable(getSecretsIfContinue(packet)).orElse(new ArrayList<>());
+      return Optional.ofNullable(getSecretsFromPacket(packet)).orElse(new ArrayList<>());
```

Comment:
We can just inline (List<V1Secret>) packet.get(SECRETS) here and remove the getSecretsFromPacket method.